Portal_Perfomance_Employes/Program.cs

```diff
@@ -1,5 +1,22 @@
+var DevelopmentSpecificOrigins = "_developmentSpecificOrigins";
+var ProductionSpecificOrigins = "_productionSpecificOrigins";
+
 var builder = WebApplication.CreateBuilder(args);
 
+builder.Services.AddCors(options =>
+{
+    options.AddPolicy(DevelopmentSpecificOrigins,
+                      policy =>
+                      {
+                          policy.WithOrigins("*").AllowAnyMethod().AllowAnyHeader();
+                      });
+    options.AddPolicy(ProductionSpecificOrigins,
+                      policy =>
+                      {
+                          policy.WithOrigins("");
```

Comment:
why nothing is added for production? should we still get some settings from config and provide it here?